### PR TITLE
Pair Games

### PR DESCRIPTION
**Pull request by Pinkuagrawal28**
## Related Issue
 
I was assigned issue number #39  to create a javascript game called pair games.

### Describe Changes

I have added all the files required for the project.

### Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 I have starred the repo
- [x] 👀 I have read the [Contributing Guideline](https://github.com/Shweta2024/JavaScript-Projects/blob/master/ContributingGuidelines.md)

### What types of changes does your code introduce?
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Screenshots
Screenshot (338)
Screenshot (339)
Screenshot (340)

### CODE REVIEW

File: Pair Games/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+1\. There are a given number of cards on the table facing down.
+
+2\. The player needs to find all the pairs.
+
+#### The Score of the game is counted in two forms:-
+
+* Time taken by the user to finish the game by matching all the pairs.
+* Number of moves required to match all the pairs
+
+### Screennshots
```
Comment (utkarsh006) [inline Pair Games/README.md:20]: Spell Check once.

**Comment by utkarsh006** [comment]
Mention issue number using # in PR template.

File: Pair Games/README.md (hunk 1 above)
Comment (Pinkuagrawal28) [reply Pair Games/README.md]: Corrected the spelling mistakes.

**Comment by Pinkuagrawal28** [comment]
> Mention issue number using # in PR template.

Done

**Comment by Shweta2024** [comment]
@Pinkuagrawal28 LGTM🎉

**Comment by Pinkuagrawal28** [comment]
Thank you😁

**Pull request merged**